backend/app/services/channels/dingtalk/handler.py

```
@@ -15,577 +24,451 @@
-                Kind.is_active == True,
-            )
-            .first()
+        super(dingtalk_stream.ChatbotHandler, self).__init__()
+        self._dingtalk_client = dingtalk_client
+        self._use_ai_card = use_ai_card
+        self._on_message = on_message
+        self._channel_id = channel_id or 0
+
+        # Handle deprecated default_team_id parameter
+        if get_default_team_id is None and default_team_id is not None:
+            get_default_team_id = lambda tid=default_team_id: tid
```

Ruff (0.14.14)

[error] Do not assign a `lambda` expression, use a `def`

Rewrite `get_default_team_id` as a `def`

(E731)